src/docs/concepts-users-and-authentication/user-permissions/index.md

```diff
@@ -14,35 +14,77 @@ permissions to join, post listings, and start transactions. However, you
 can modify some of these permissions in Console, under the "Access
 control" tab.
 
+![Access control options](./access_control_blank.png)
+
+## Make marketplace private
+
+By default, Sharetribe marketplaces are public. This means that listings
+and user profiles are visible to unauthenticated users. By making your
+marketplace private, you can allow only authenticated users to view
+listings and users on your marketplace.
+
+On a private marketplace, the only public Marketplace API endpoints are
+user creation related [[ TODO CHECK IF THIS IS TRUE ]]. All other
+marketplaces require an authenticated user access token.
+
+This setting only has options on the marketplace level.
```

Comment:
This sentence is a bit ambiguous, I'm not fully sure what this means here 🤔

Reply:
The other permissions features have a marketplace level toggle and a user level toggle, but this one can only be turned on or off on the marketplace level. Updating the wording here!